storagenode/operator/operatorserver/server.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package operatorserver
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"strings"
+
+	"github.com/prometheus/common/log"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+	"golang.org/x/sync/errgroup"
+
+	"storj.io/storj/storagenode/operator"
+)
+
+// TODO: improve embedded resources generation
+//go-bindata -pkg operatorserver -o storagenode/operator/operatorserver/static.go web/operator/dist web/operator/dist/public/
```

Comment:
I like that soo much!

Reply:
This could also be a Make Target. that we add to our release process.
That should then include the most recent assets into our binary.

Reply:
Oh, nooo... this will bloat our github repository. I've seen repositories end up as 2GB as an end result.

Reply:
@egonelbre How do you want to install and release binaries without that?

Reply:
Created an issue to discuss this: https://storjlabs.atlassian.net/browse/V3-2002

Reply:
@egonelbre truly to say - we have an SPA with minified js bundle ~ 1mb and a few mb of media resources, that also could be minified with FFMPEG. I don't think that we will have more that 5mb of data

@stefanbenten from the other side - storagenode operator - is a local server. I don't think that we will have problems with not embedded static files there

Reply:
This will be eventually committed 1000x or more times and minified files to compress.. so if 5MB it'll end up as 5GB.

Reply:
@egonelbre 
By the way - i've just checked and all assets and js from deployed satellite console weights ~1.8 mb
Node operator will be much more lightweight

Reply:
i agree with egon - if we check in bundles that are generated from the code, then every time we change a line in the relevant parts of the code it's a new couple megabytes. it does balloon quickly